cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -292,7 +292,8 @@ func (s *CMServer) Run(_ []string) error {
 	}
 
 	if s.AllocateNodeCIDRs {
-		if cloud == nil {
+		// TODO: Pipe this as a command line flag that corresponds to overlay==true
+		if cloud == nil || true {
```

Comment:
a todo, it prevents route creation. we have no way to turn it off today on gce today and I need to pipe a flag that's set when we start the cluster (network-provier=flannel should set this flag).